src/DetailsView/components/details-view-switcher-nav.ts

```diff
@@ -41,17 +49,20 @@ export type GetDetailsSwitcherNavConfigurationProps = {
 
 const detailsViewSwitcherNavs: { [key in DetailsViewPivotType]: InternalDetailsViewSwitcherNavConfiguration } = {
     [DetailsViewPivotType.assessment]: {
-        CommandBar: CommandBarWithExportAndStartOver,
+        CommandBar: AssessmentCommandBar,
```

Comment:
Hm, so I think we're effectively coupling the pivot type with the export button twice. Once here and another time inside of the specific command bars.

I think it'd be better to maybe move the methods to deps and then the command bars decided which one to use (eliminating the need to specify them here).

Reply:
The only difference now between AutomatedChecksCommandBar and AssessmentChecksCommandBar is the NamedFC. There is still an element of double coupling--let's discuss the value of the NamedFC distinction. If it's unimportant, then CommandBar can just be DetailsViewCommandBar and we're good to go